samples/deployment/test-network/README.md

```diff
@@ -189,6 +189,19 @@ cd $FPC_PATH/samples/deployment/test-network/fabric-samples/test-network
 ./network.sh down
 ```
 
+## Changing chaincode used by test-network sample
+If you have already reached this point you can try changing the chaincode used by FPC. For example, you can use the HelloWorld code instead of the echo code. To do so you have to change the values of the environment variables set at the beginning `CC_ID` and `CC_PATH`.
```

Comment:
Here we should link to the other tutorial. Please use a relative markdown link.

Reply:
What about also adding a reference to this section in the hello world tutorial?

Reply:
> If you have already reached this point you can try changing the chaincode used by FPC.

Maybe .... 
"In this section we show how you can run a different FPC chaincode on the test network. For example, ..."

Reply:
Changed text.